routing/pathfind.go

```diff
@@ -1192,14 +1193,11 @@ type blindedPathRestrictions struct {
 	// nodeOmissionSet holds a set of node IDs of nodes that we should
 	// ignore during blinded path selection.
 	nodeOmissionSet fn.Set[route.Vertex]
-}
 
-// blindedHop holds the information about a hop we have selected for a blinded
-// path.
-type blindedHop struct {
-	vertex       route.Vertex
-	channelID    uint64
-	edgeCapacity btcutil.Amount
+	// incomingChainedChannels holds the chained channels list specified
+	// via scid (short channel id) starting from a channel which points to
+	// the receiver node.
+	incomingChainedChannels []uint64
```

Comment:
That seems strange, we provide two times almost the same information to the findBlindedRoute function ? Here and in partiallyChainedPaths ? Overlooked that.